### PR TITLE
Add production build script to Travis

### PR DESCRIPTION
**Pull request by kristandre**
Create shell script to check if build is completed without error.
Build without --progress

### CODE REVIEW

File: travis_build_prod.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+echo "Building production files while checking for errors..."
+
+if npm run build:prod:travis | grep ERROR
+then
+   echo "Build FAILED!"
+   exit 1
+else
+   echo "Build was successful!"
+   exit 0
+fi
```
Comment (NorwegianKiwi) [inline travis_build_prod.sh]: Legger du på et linjeskift på siste linje?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - "6.2.2"
+before_install:
+  - git clone https://github.com/kodeklubben/oppgaver.git ../oppgaver
 script:
+  - ./travis_build_prod.sh
   - npm run eslint
   - npm run test
```
Comment (NorwegianKiwi) [inline .travis.yml]: Legger du på et linjeskift på siste linje?

File: package.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "buildstatic": "webpack --config=webpack.static.config.babel.js --progress",
     "build": "npm run buildjs && npm run buildstatic",
     "build:prod": "npm run buildjs:prod && npm run buildstatic",
+    "build:prod:travis": "cross-env NODE_ENV=production webpack -p && webpack --config=webpack.static.config.babel.js",
```
Comment (arve0) [inline package.json]: Fungerer && på Windows?

Comment (kristandre) [inline package.json]: Fra CMD doc:
> command1 && command2
Use to run the command following && only if the command preceding the symbol is successful. Cmd.exe runs the first command, and then runs the second command only if the first command completed successfully.

**Pull request merged**